internal/ui/input_glfw.go

```diff
@@ -78,7 +87,18 @@ var glfwMouseButtonToMouseButton = map[glfw.MouseButton]MouseButton{
 func (i *Input) update(window *glfw.Window, scale float64) {
 	i.m.Lock()
 	defer i.m.Unlock()
-
+	if i.runebuffer == nil {
+		i.runebuffer = make([]rune, 0, 1024)
+		window.SetCharModsCallback(func(w *glfw.Window, char rune, mods glfw.ModifierKey) {
+			if unicode.IsPrint(char) {
```

Comment:
I see. Well, we needed to consider more actual use cases before implementing a new feature. I'm still not sure if it is fine to treat only printable characters, but leave this as it is so far.